.docker/Dockerfile.ckan

```diff
@@ -22,22 +22,18 @@ RUN . /app/ckan/default/bin/activate \
     && deactivate \
     && ln -s /app/ckan /usr/lib/ckan
 
-COPY docker.ini /app/ckan/default/production.ini
+COPY .docker/test.ini /app/ckan/default/production.ini
```

Comment:
removed duplicated `test.ini` and renamed `docker.ini` to `test.ini` in order to remove confusion between 2 configs used for the same purpose.

Also, moved it under `.docker` since it is only used in docker.